.github/workflows/release.yml

```diff
@@ -5,19 +5,197 @@ on:
   workflow_dispatch:
 
 jobs:
+  create-release:
+    name: Create Release
+    runs-on: ubuntu-20.04
+    outputs:
+      mobile-version: ${{ steps.retrieve-mobile-version.outputs.mobile_version }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f  # v2.3.4
+        with:
+          ref: rc
+
+      - name: Retrieve Mobile release version
+        id: retrieve-mobile-version
+        run: |
+          ver=$(sed -n -e '/android:versionName/ s/.*\= *//p' ./src/Android/Properties/AndroidManifest.xml | tr -d '"')
+          echo "::set-output name=mobile_version::${ver}"
+        shell: bash
+
+      - name: Check to make sure Mobile release version has been bumped
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          latest_ver=$(hub release -L 1 -f '%T')
+          latest_ver=${latest_ver:1}
+          echo "Latest version: $latest_ver"
+          ver=${{ steps.retrieve-mobile-version.outputs.mobile_version }}
+          echo "Version: $ver"
+          if [ "$latest_ver" = "$ver" ]; then
+            echo "Version has not been bumped!"
+            exit 1
+          fi
+        shell: bash
+
+      - name: Create Draft Release
+        uses: actions/create-release@0cb9c9b65d5d1901c1f53e5e66eaf4afd303e70e  # v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: v${{ steps.retrieve-mobile-version.outputs.mobile_version }}
+          release_name: Version ${{ steps.retrieve-mobile-version.outputs.mobile_version }}
+          draft: true
+          prerelease: false
+
+
+  attach-artifacts:
+    name: Attach Release Artifacts
+    runs-on: windows-2019
+    needs: create-release
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f  # v2.3.4
+        with:
+          ref: rc
+
+      - name: Download Play Store .aab artifact
+        uses: dawidd6/action-download-artifact@b9571484721e8187f1fd08147b497129f8972c74  # v2.14.0
+        with:
+          workflow: build.yml
+          workflow_conclusion: success
+          branch: rc
+          name: com.x8bit.bitwarden.aab
```

Comment:
> From documentation of [action-download-artifact](https://github.com/dawidd6/action-download-artifact)
    # Optional, uploaded artifact name,
    # will download all artifacts if not specified
    # and extract them in respective subdirectories
    # https://github.com/actions/download-artifact#download-all-artifacts
    name: artifact_name

Thoughts on removing the name and letting it download all of the artifacts?

Reply:
Saves some lines.  I changed it to download all of them in one step.